cypress/e2e/views/common.view.ts

```diff
@@ -96,3 +96,7 @@ export enum sideDrawer {
 }
 export const closeAbout = "button[aria-label='Close Dialog'";
 export const pencilIcon = "#pencil-action";
+export function getCheckboxSelector(text: string) {
+    text = text.charAt(0).toLowerCase() + text.slice(1).replace(/\s+/g, "");
+    return `input[aria-labelledby='check-${text}']`;
+}
```

Comment:
I think this belongs to the `utils` file as this file contains consts only

Reply:
Done!